src/sites/iva/iva-app.js

```diff
@@ -1601,7 +1601,9 @@ class IvaApp extends LitElement {
 
                 ${this.config.enabledComponents.account ? html`
                     <div class="content" id="account">
-                        <user-profile .opencgaSession="${this.opencgaSession}"></user-profile>
+                        <user-profile .opencgaSession="${this.opencgaSession}"
+                                      .settings="${USER_PROFILE_SETTINGS}">
+                        </user-profile>
```

Comment:
Please reformat this moving all attributes to a new line and using only 4 spaces for indentation:

```html
<user-profile
    .opencgaSession="${this.opencgaSession}"
    .settings="${USER_PROFILE_SETTINGS}">
</user-profile>
```

Reply:
Where is this documented as code-style guidance?

Reply:
This is not documented yet, the style guide is not finished, sorry. We are instantiating all new components using that style and migrating as many as we can. I hope that style is fine for you.